index.js

```diff
@@ -67,6 +61,12 @@ HtmlWebpackPlugin.prototype.apply = function(compiler) {
               assets.favicon = faviconBasename;
             });
         }
+        if (self.options.manifest) {
```

Comment:
This isn't necessary as you can use a webpack app cache plugin - like [appcache-webpack-plugin](https://github.com/lettertwo/appcache-webpack-plugin) will add a `.appcache` to the resources and will be picked up by the html-webpack-plugin: https://github.com/ampedandwired/html-webpack-plugin/blob/feature/loaders/index.js#L349